app/views/govuk_publishing_components/components/_inverse_header.html.erb

```diff
@@ -1,10 +1,12 @@
 <%
   full_width ||= false
   header_class = full_width ? "gem-c-inverse-header--full-width" : ""
+  no_padding_top ||= false
```

Comment:
I think this is fractionally more confusing than it could be. Could the variable be just 'padding_top', and adjust the logic accordingly?

Reply:
Agreed - I put this up for review then realised it was a bit backwards. Will change :+1: